gridt/src/app/core/user.model.ts

```diff
@@ -13,5 +13,8 @@ export interface User {
         message?: string
     } | null;
 
+    /**
+     * The URL linking to the user's profile picture on gravatar.com
+     */
     avatar: string;
```

Comment:
Currently the server does not respond with avatars yet, so maybe add a question mark, as to indicate that this isn't there yet?